pilot/pkg/networking/plugin/authn/authentication.go

```diff
@@ -128,9 +136,15 @@ func ConvertPolicyToJwtConfig(policy *authn.Policy) *jwtfilter.JwtAuthentication
 		AllowMissingOrFailed: true,
 	}
 	for _, policyJwt := range policyJwts {
-		hostname, port, _, err := model.ParseJwksURI(policyJwt.JwksUri)
+		jwksURI, err := getJwksURI(policyJwt)
+		if err != nil {
+			log.Errorf("Cannot get jwks_uri for issuer %q: %v", policyJwt.Issuer, err)
```

Comment:
I've just learned that we should mostly use Warnf for non-critical error. Can you change this and other errorf.

Reply:
done